book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Wouldn't this method add duplicates of a rental into the `rentals` variable. For example, we need a book object to create a rental (in the rental) class. If we call this method with a rental object, `rentals.push(rental)` will make it duplicate (see rentals count in image).

  To avoid duplicates, why don't we make this method return a Rental instantiation like in the code snippet below.
  **NOTE:** You will need to import the Rental class here
  ```
  def add_rental(date, person)
    Rental.new(date, self, person)
  end
  ```
  ![mahdi-2](https://user-images.githubusercontent.com/36542986/175008989-532944b0-9cff-4bd9-a6cb-ec655574be4a.jpg)